client/signup/steps/design-type-with-atomic-store/style.scss

```diff
@@ -1,193 +0,0 @@
-.design-type-with-atomic-store {
```

Comment:
I love all this red

Reply:
First we copy it and then we delete it 💯